format/spec.md

```diff
@@ -1128,15 +1128,15 @@ This serialization scheme is for storing single values as individual binary valu
 | **`long`**         | **`JSON long`**                           | `34`                                       | |
 | **`float`**        | **`JSON number`**                         | `1.0`                                      | |
 | **`double`**       | **`JSON number`**                         | `1.0`                                      | |
-| **`decimal(P,S)`** | **`JSON number`**                         | `14.20`                                    | Stores the decimal value as a general json number |
+| **`decimal(P,S)`** | **`JSON number`**                         | `14.20`                                    | Stores the decimal as a number with S places after the decimal |
 | **`date`**         | **`JSON string`**                         | `"2017-11-16"`                             | Stores ISO-8601 standard date |
 | **`time`**         | **`JSON string`**                         | `"22:31:08.123456"`                        | Stores ISO-8601 standard time with microsecond precision |
 | **`timestamp`**    | **`JSON string`**                         | `"2017-11-16T22:31:08.123456"`             | Stores ISO-8601 standard timestamp with microsecond precision; must not include a zone offset |
 | **`timestamptz`**  | **`JSON string`**                         | `"2017-11-16T22:31:08.123456-07:00"`       | Stores ISO-8601 standard timestamp with microsecond precision; must include a zone offset |
-| **`string`**       | **`JSON string`**                         | `"foo"`                                    | |
-| **`uuid`**         | **`JSON string`**                         | `"eb26bdb1-a1d8-4aa6-990e-da940875492c"`   | Stores the lowercase uuid string |
-| **`fixed(L)`**     | **`JSON string`**                         | `"0x3162"`                                 | Stored as a hexadecimal string, prefixed by `0x` |
-| **`binary`**       | **`JSON string`**                         | `"0x3162"`                                 | Stored as a hexadecimal string, prefixed by `0x` |
+| **`string`**       | **`JSON string`**                         | `"iceberg"`                                | |
+| **`uuid`**         | **`JSON string`**                         | `"f79c3e09-677c-4bbd-a479-3f349cb785e7"`   | Stores the lowercase uuid string |
+| **`fixed(L)`**     | **`JSON string`**                         | `"0x00010203"`                             | Stored as a hexadecimal string, prefixed by `0x` |
```

Comment:
Should we also note that the hex string length should be less than or equal to 2*L

Reply:
Shouldn't it be exactly `2L+2`?

Reply:
Oh I was talking about the part after the `0x`, I was previously thinking if less than `2L + 2` we prepend `0`s to it. But I also think making it strictly `2L+2` works too.

Reply:
I don't think there's a need to specify the length. I just wanted to clarify that it will never be `less than or equal to 2L`. It will always be `2L+2`.